charts/mattermost-operator/templates/mattermost-operator/deployment.yaml

```diff
@@ -0,0 +1,38 @@
+{{- if .Values.mattermostOperator.enabled }}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ template "mattermost-operator.name" . }}
+  namespace: {{ template "mattermost-operator.namespace" . }}
+  labels:
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/name: {{ template "mattermost-operator.name" . }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+spec:
+  replicas: {{ .Values.mattermostOperator.replicas }}
+  selector:
+    matchLabels:
+      name: {{ template "mattermost-operator.name" . }}
```

Comment:
the above labels should be applied here as well

Reply:
Thanks my friend, nice catch, done :)